src/test/validation_block_tests.cpp

```diff
@@ -88,22 +88,27 @@ std::shared_ptr<CBlock> MinerTestingSetup::Block(const uint256& prev_hash)
     txCoinbase.vout[1].nValue = txCoinbase.vout[0].nValue;
     txCoinbase.vout[0].nValue = 0;
     txCoinbase.vin[0].scriptWitness.SetNull();
+    txCoinbase.vin[0].scriptSig = WITH_LOCK(cs_main, return CScript() << LookupBlockIndex(prev_hash)->nHeight + 1 << OP_0);
```

Comment:
Could add comment like `    // Always pad with OP_0 at the end to avoid bad-cb-length error`